src/data_warehouse/cli/deploy.py

```
@@ -0,0 +1,154 @@
+    If no version is specified, the latest version will be used.
+    """
+    target_env = environment or settings.ENVIRONMENT
+    version_txt = version or "latest"
+
+    click.echo(
+        f"Upgrading deployment in {target_env} environment to version {version_txt}..."
+    )
+
+    # Log the details
+    logger.info(
+        f"Deployment upgrade started - Environment: {target_env}, Version: {version_txt}"
```

Ruff (0.8.2)

Line too long (89 > 88)

(E501)